scripts/trainer.py

```diff
@@ -37,15 +37,7 @@ def go(self, model_name):
         logger.info("Decompressing bugs database")
         self.decompress_file("data/bugs.json")
 
-        log_messages = {
-            "component": "Training *component* model",
-            "defectenhancementtask": "Training *defect vs enhancement vs task* model",
-            "regression": "Training *regression vs non-regression* model",
-            "training": "Training *tracking* model",
-        }
-
-        if log_messages.get(model_name):
-            logger.info(log_messages[model_name])
+        logger.info(f"Training *{model_name}* name")
```

Comment:
s/name/model :P

Reply:
(my fault, I see I used the wrong string in my comment earlier)

Reply:
So this should be logger.info(f"Training *{model_name}* model")

Reply:
It should be fixed now.

I should have checked it before pasting.